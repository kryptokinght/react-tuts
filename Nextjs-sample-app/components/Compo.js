const Compo = () => {
    return (
        <div>
            Hello World
        </div>
    );
}

export default Compo;