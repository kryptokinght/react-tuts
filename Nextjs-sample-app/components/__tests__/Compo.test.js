import React from 'react';
import Compo from '../Compo';
import { render, shallow } from 'enzyme';

describe('<Compo />', () => {
    it('should be defined', () => {
        expect(Compo).toBeDefined();
    });
    it('should render', () => {
        const wrapper = shallow(<Compo />);
        expect(wrapper).toMatchSnapshot();
    });
});