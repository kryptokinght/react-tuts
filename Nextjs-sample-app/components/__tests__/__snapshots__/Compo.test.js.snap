// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Compo /> should render 1`] = `
<div>
  Hello World
</div>
`;
